service/service_webhook_test.go

```diff
@@ -0,0 +1,188 @@
+package service
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"net/http/httptest"
+	"strings"
+	"testing"
+
+	"github.com/Sirupsen/logrus"
+	"github.com/mitchellh/mapstructure"
+	v1client "github.com/rancher/go-rancher/client"
+	"github.com/rancher/go-rancher/v2"
```

Comment:
Use `gocheck` for unit test.

Also adds a blank line before `github.com/rancher/webhook-service` to separate current package with others.

Reply:
You haven't updated this part.